pkg/neotest/compile.go

```diff
@@ -90,3 +92,30 @@ func CompileFile(t testing.TB, sender util.Uint160, srcPath string, configPath s
 	contracts[cacheKey] = c
 	return c
 }
+
+// LoadContract loads a contract from the specified NEF and manifest files.
+func LoadContract(t testing.TB, sender util.Uint160, nefPath, manifestPath string) (*Contract, error) {
+	nefBytes, err := os.ReadFile(nefPath)
+	require.NoError(t, err)
+
+	ne, err := nef.FileFromBytes(nefBytes)
+	require.NoError(t, err)
+
+	manifestBytes, err := os.ReadFile(manifestPath)
+	require.NoError(t, err)
+
+	m := new(manifest.Manifest)
+	err = json.Unmarshal(manifestBytes, m)
+	require.NoError(t, err)
+
+	err = m.IsValid(util.Uint160{}, true)
+	require.NoError(t, err)
+
+	hash := state.CreateContractHash(sender, ne.Checksum, m.Name)
+
+	return &Contract{
```

Comment:
You may use `sender.StringLE() | nefPath | manifestPath` as the cache key, and then add this contract to the cache.